snuba/utils/streams/topics.py

```diff
@@ -18,5 +18,8 @@ class Topic(Enum):
 
 
 def get_topic_creation_config(topic: Topic) -> Mapping[str, str]:
-    config = {Topic.EVENTS: {"message.timestamp.type": "LogAppendTime"}}
+    config = {
+        Topic.EVENTS: {"message.timestamp.type": "LogAppendTime"},
+        Topic.METRICS: {"message.timestamp.type": "LogAppendTime"},
```

Comment:
@lynnagara was this all I needed?

Reply:
Yes, thanks. Has this topic already being created in production? If yes we should make sure it has this configuration, if no we need to create it with this configuration.

Reply:
@lynnagara 
> Has this topic already being created in production?
 not to my knowledge, is this something we can open a PR for? or is it a request to ops?

Reply:
You can wait till when we will run the consumer in prod. Anyway there is no Clickhouse storage. Let's get everything working properly in dev first